.github/dependabot.yml

```diff
@@ -9,6 +9,7 @@ updates:
   schedule:
     interval: "daily"
   commit-message:
+    prefix: "dependabot:"
```

Comment:
/lgtm cancel

Reply:
if you were looking at `:seedling:` as reference it has semicolons on both sides because that's the syntax for emojis in markdown

Reply:
Yeah, I purposefully put `dependabot:` to get a commit message of type `dependabot: updated something`.

Do we want to use some other prefix?

Reply:
or I can just remove the `:`

Reply:
I'm assuming that dependabot will include the semicolon for us between prefix and commit message so `prefix: dependabot:` would result in a commit message like `dependabot:: updated dependency xyz` but maybe I'm wrong

Reply:
Updated and removed the semicolon for now :)